### PR TITLE
Adjustments for API changes in membrane_ice_plugin and ex_dtls

### PR DESCRIPTION
**Pull request by mickel8**
related with:
membraneframework/membrane_ice_plugin#21
membraneframework/ex_dtls#8

### CODE REVIEW

File: lib/handshake.ex [hunk 1]
```diff
@@ -16,27 +16,33 @@ defmodule Membrane.DTLS.Handshake do
...
   @impl Handshake
-  def recv_from_peer(%{dtls: dtls}, data) do
-    ExDTLS.do_handshake(dtls, data)
+  def process(data, %{dtls: dtls}) do
+    ExDTLS.process(dtls, data)
+  end
+
+  @impl Handshake
+  def is_hsk_packet(<<head, _rest::binary()>> = packet, _state) do
+    19 < head and head < 64 and byte_size(packet) >= 13
```
Comment (mat-hek) [inline lib/handshake.ex]:
```suggestion
    head in 20..63 and byte_size(packet) >= 13
```

File: lib/handshake.ex [hunk 2]
```diff
@@ -16,27 +16,36 @@ defmodule Membrane.DTLS.Handshake do
   require Membrane.Logger
 
   @impl Handshake
-  def init(opts) do
+  def init(id, parent, opts) do
```
Comment (mat-hek) [inline lib/handshake.ex]: it seems the new arguments are never used

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: lib/handshake.ex (hunk 2 above)
Comment (mickel8) [reply lib/handshake.ex]: Yeah, they are prepared for future retransmissions support. We will use them to send `{:retransmit, id, packet}` message to ICE

Comment (mat-hek) [reply lib/handshake.ex]: then just ignore them for now

**Review by mat-hek — APPROVED** [review]

**Pull request merged**